src/features/gameboard/events/event-definitions/edging/cum.ts

```diff
@@ -1,23 +1,24 @@
-import { GameEvent, EGrip } from '../../../types'
+import { GameEvent, EGrip, PlayerParts } from '../../../types'
 import { GameBoardActions } from '../../../store'
 import { MessageType } from '../../../MessageArea/MessageTypes'
 import { playTone } from '../../../sound'
 import { wait } from '../../helpers'
 
 export const cum: GameEvent = () => {
   return (state, dispatch) => {
+    let hasCock = state.settings.player.parts === PlayerParts.Cock
```

Comment:
Nit pick: since this lives only in this closure and shouldn't be changed during a game tick, it should be a `const` here

```suggestion
    const hasCock = state.settings.player.parts === PlayerParts.Cock
```